nc_time_axis/tests/unit/test_NetCDFTimeConverter.py

```diff
@@ -3,13 +3,17 @@
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
+import sys
 import unittest
 
 import cftime
 import numpy as np
 
 from nc_time_axis import NetCDFTimeConverter, CalendarDateTime
 
+if sys.version_info[:2] == (2, 7):
+    unittest.TestCase.assertRaisesRegex = unittest.TestCase.assertRaisesRegexp
+
```

Comment:
@ocefpaf Could we not just use `from six import assertRaisesRegex` instead?

Reply:
Sure. Let me update that.